### PR TITLE
Updated OpenVINO docs for Aurora

### PR DESCRIPTION
**Pull request by rickybalin**
Updated OpenVINO docs for Aurora. 

Instructions for using OpenVINO C++ API were temporarily removed due to some issues building the C++ example, but these will be added back in the future.

### CODE REVIEW

**Comment by felker** [comment]
you might consider leaving the `## Inference with C++ OpenVINO API` etc section in and surrounding it with HTML comments

File: docs/aurora/data-science/frameworks/openvino.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Model Inference with OpenVINO
+
+OpenVINO is a library developed by Intel specifically designed for accelerating inference of ML models on their CPU and GPU hardware. 
+This page contains build and run instructions for Python, but please refer to the [OpenVINO Github page](https://github.com/openvinotoolkit/openvino) for more information.
```
Comment (felker) [inline docs/aurora/data-science/frameworks/openvino.md:4]:
```suggestion
This page contains build and run instructions for Python, but please refer to the [OpenVINO GitHub page](https://github.com/openvinotoolkit/openvino) for more information.
```

File: docs/aurora/data-science/frameworks/openvino.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+OpenVINO does not come with the default frameworks module on Aurora, but it can be installed manually within a Python virtual environment as shown below
+```bash
+module load frameworks/2024.2.1_u1
+python -m venv --clear /path/to/_ov_env --system-site-packages
+source /path/to/_ov_env/bin/activate
+pip install openvino==2024.4.0
+pip install openvino-dev==2024.4.0
+```
+
+It is recommended that the path to the virtual env be in the user's project space on Flare.
```
Comment (felker) [inline docs/aurora/data-science/frameworks/openvino.md:18]:
```suggestion
It is recommended that the path to the virtual env be in the user's project space on [Flare](../../data-management/lustre/flare/).
```

**Comment by felker** [comment]
LGTM. Ready to merge @rickybalin or do you want to re-add the removed C++ section surrounded in HTML quotes? Also could replace \``` with ```c++

**Comment by rickybalin** [comment]
Thanks, @felker . Yes, I'll add the C++ section with comments update the PR.

File: docs/aurora/data-science/frameworks/openvino.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+OpenVINO does not come with the default frameworks module on Aurora, but it can be installed manually within a Python virtual environment as shown below
+```bash
+module load frameworks/2024.2.1_u1
+python -m venv --clear /path/to/_ov_env --system-site-packages
+source /path/to/_ov_env/bin/activate
+pip install openvino==2024.4.0
+pip install openvino-dev==2024.4.0
+```
+
+It is recommended that the path to the virtual env be in the user's project space on [Flare](../../data-management/lustre/flare/).
```
Comment (felker) [inline docs/aurora/data-science/frameworks/openvino.md:18]:
```suggestion
It is recommended that the path to the virtual env be in the user's project space on [Flare](../../data-management/lustre/flare.md).
```

**Comment by rickybalin** [comment]
@felker I added the C++ example back commented out. Should be good to merge now.

**Review by felker — APPROVED** [review]

**Pull request merged**